### DISCUSSION TITLE
I wrote an article on Renovate

### DISCUSSION THREAD

**Discussion by onigoetz** [renovatebot/renovate#16484] [Suggest an Idea]
Hi,

just a heads up that I wrote an article on how we at Swissquote ( a Swiss online bank ) use Renovate to keep our dependencies up-to-date : https://medium.com/swissquote-engineering/how-swissquote-is-keeping-software-dependencies-up-to-date-with-renovate-6246e8b20437

thanks for everything you do and keep up the good work 😄

**Comment by rarkins** [comment]
Thanks for sharing! That's one of the best articles about using Renovate which anyone's ever written

**Comment by HonkingGoose** [comment]
Hi @onigoetz! 👋

I'm impressed! Your article is fun, easy-to-read, and has good tips and advice. 😍

I'm thinking of creating a section in our docs called `User Stories` to highlight Renovate users experiences, stories, and tips. Your article would be a great start to this new section! 😄 Here's the big question: do you want to put your article into our official documentation?

If you agree, here are the steps to get your article in our docs:

1. You copy/paste the content into `docs/usage/user-stories/swissquote.md` and:
    - Put your name and contact information at the top of the article
    - _Optional:_ link to your personal blog, or GitHub profile
    - _Optional:_ link to company homepage
    - _Optional:_ link to the post on Medium
1. You create a PR, so you have Git commit authorship.
1. Ping me to review the PR.
1. After _your_ PR is merged, I'll create a PR to put your content on our docs sidebar.

Let me know what you decide. 😉

Greetings,

@HonkingGoose

---

This discussion is resolved with this PR:

-  https://github.com/renovatebot/renovate/pull/16809

**Reply by onigoetz** [reply]
Thanks for the kind words :)

I'd like that very much! 

Two little things though : 
1) Since it would be the same content, how would it work for indexers and canonical references ?
2) I'm off for a month right now, so I won't always be by a computer and might do it around mid august if that's ok with you.

Best regards,

@onigoetz

**Reply by HonkingGoose** [reply]
Awesome! 🥳🎉

> 1. Since it would be the same content, how would it work for indexers and canonical references ?

I think you're worried about search ranking for your Medium article? I don't know anything about indexers or canonical references, I'll let one of the Renovate developers answer this. 😉 

> 2. I'm off for a month right now, so I won't always be by a computer and might do it around mid august if that's ok with you.

That's fine by me! I'll create an issue to track this work. 😉 

- #16502

**Reply by rarkins** [reply]
I'm not sure that mkdocs (which we use for documentation publishing) can set a canonical path *per page*, but I think we could include "This article was originally published at https:..." and you'd hope Google is smart enough to treat the one we link to as the canonical source and not ours

**Reply by HonkingGoose** [reply]
> I'm not sure that mkdocs (which we use for documentation publishing) can set a canonical path per page

We can only set it the canonical URL for the _whole_ website. Quote from the `site_url` [^mkdocs] docs:

> ### `site_url`
> 
> Set the canonical URL of the site. This will add a `link` tag with the
> `canonical` URL to the `head` section of each HTML page. If the 'root' of the
> MkDocs site will be within a subdirectory of a domain, be sure to include that
> subdirectory in the setting (`https://example.com/foo/`).
> 
> This setting is also used for `mkdocs serve`: the server will be mounted onto a
> path taken from the path component of the URL, e.g. `some/page.md` will be
> served from `http://127.0.0.1:8000/foo/some/page/` to mimic the expected remote
> layout.

Here's the `site_url` setting in our `mkdocs.yml` file: [^config-file]

```yaml
# Set the canonical URL of the site.
# https://www.mkdocs.org/user-guide/configuration/#site_url
site_url: 'https://docs.renovatebot.com'
```

> but I think we could include "This article was originally published at https:..." and you'd hope Google is smart enough to treat the one we link to as the canonical source and not ours

Sounds good to me.

[^mkdocs]: [MkDocs documentation, `site_url`](https://www.mkdocs.org/user-guide/configuration/#site_url)
[^config-file]: https://github.com/renovatebot/renovatebot.github.io/blob/21713468c38975c1edb7e075d0cc5b753e947fab/mkdocs.yml#L20-L22

**Reply by viceice** [reply]
maybe canonical page link will be possible with a mkdocs hook, like we already use for edit page links
